### PR TITLE
How to guide for preparation of the prezipped dataset files

### PR DESCRIPTION
**Pull request by cmadjar**
This adds an how to guide for the preparation of the prezipped dataset files that will be available for download.

### CODE REVIEW

File: Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+```
+datalad install <dataset name>
+```
+
+Notes:
+ 
+- this step does not need to be done on the production server since the whole `conp-dataset`
+has been installed recursively
+- for derived dataset, need to install the `conp-dataset` temporarily elsewhere without the `-r`
```
Comment (xlecours) [inline Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md:38]:
```suggestion
- for derived dataset, we need to install the `conp-dataset` temporarily elsewhere without the `-r` 
```

**Comment by xlecours** [comment]
I am lukewarm point#5. Other then that, it looks all right.

**Comment by cmadjar** [comment]
@xlecours @emmetaobrien I finalized this documentation. Those are the steps I used to create all the prezipped archives that I just copied on the portal VM. 

Ready for final review :).

**Comment by cmadjar** [comment]
@xlecours @emmetaobrien any chance you could review this next week? Thank you!

**Comment by xlecours** [comment]
Again, the `git log` part seems accessorial. Other than that it looks all right.

**Review by xlecours — CHANGES_REQUESTED** [review]
Again, the `git log` part seems accessorial. Other than, since the naming of the dataset with the `_version-` part is primordial for the portal download button, it should be mentionned.

File: Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+##### 8. Add the dataset version to the archive filename
+
+This was done by hand but could be scripted in future creation of prezipped
+archives.
+
+```
+mv <dataset name>.tar.gz <dataset_name>_version-<version-number>.tar.gz
+```
```
Comment (cmadjar) [inline Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md:149]: @xlecours Not sure I understand your latest comment. The renaming with the proper version is done in step 8. here. Am I missing something?

Comment (cmadjar) [reply Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md:156]: @xlecours I just pushed some changes that maybe will reply to your comments about he _version- part?

Comment (xlecours) [reply Developers-Notes/conp-dataset_creation_of_prezipped_archives_for_downloads.md:156]:
```suggestion
mv <dataset name>.tar.gz <dataset_name>_version-<version-string>.tar.gz
Where <version-string> is the value of the version property in the DATS.json file.
```

**Review by xlecours — CHANGES_REQUESTED** [review]

**Comment by cmadjar** [comment]
@xlecours thanks for the suggestion! Done!

**Pull request merged**